scm-plugins/scm-svn-plugin/src/test/java/sonia/scm/repository/spi/SvnModifyCommandTest.java

```diff
@@ -191,4 +194,54 @@ public void shouldPassIfExpectedRevisionMatchesCurrentRevision() throws IOExcept
 
     // nothing to check here; we just want to ensure that no exception is thrown
   }
+
+  @Test
+  public void shouldFailIfLockedByOtherPerson() {
+    Subject subject = mock(Subject.class);
+    when(subject.getPrincipal()).thenReturn("Perrin");
+    ThreadContext.bind(subject);
+
+    lockFile();
+
+    initSecurityManager();
+
+    ModifyCommandRequest request = new ModifyCommandRequest();
+    request.addRequest(new ModifyCommandRequest.DeleteFileRequest("a.txt", false));
+    request.setCommitMessage("this should not happen");
+    request.setAuthor(new Person("Arthur Dent", "dent@hitchhiker.com"));
+
+    assertThrows(FileLockedException.class, () -> svnModifyCommand.execute(request));
+    WorkingCopy<File, File> workingCopy = workingCopyFactory.createWorkingCopy(context, null);
+    assertThat(new File(workingCopy.getWorkingRepository().getAbsolutePath() + "/a.txt")).exists();
```

Comment:
`ThreadContext.unbindSubject()`?

Reply:
This is done in `cleanUpSecurityManager` which is annotated with `@After`, or am I missing something?

Reply:
You are right, i missed this one in the diff view